Gopkg.lock

```diff
@@ -3,107 +3,140 @@
 
 [[projects]]
   branch = "master"
+  digest = "1:08dcd5fd9153189d126eb0634d512263ab26dfd27a0e5850db10738bca908196"
   name = "github.com/Scalingo/envconfig"
   packages = ["."]
+  pruneopts = "NUT"
   revision = "8eb48cc6f27eafda8e3a9627edba494a1d229a01"
 
 [[projects]]
   branch = "master"
+  digest = "1:e25669f49084df10ec9ff820eb625cfd90a136331bfb79112f88307f7829ad9f"
   name = "github.com/Scalingo/go-gitremote"
   packages = ["."]
+  pruneopts = "NUT"
   revision = "97b505dac0c3696d7395d6627494ec105a69280f"
 
 [[projects]]
+  branch = "fix/cli/344/router_settings"
```

Comment:
User the master branch

Reply:
Then merge https://github.com/Scalingo/go-scalingo/pull/71 :p